backend/controllers/orderController.js

```
@@ -179,8 +367,132 @@
+  </div>
+</body>
+</html>
+  `,
+      };
+
+      await transporter.sendMail(mailOptions, (error, info) => {
+        if (error) {
+          console.log("Error: ", error);
+        } else {
+          console.log("Email sent: " + info.response);
+        }
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)